tests/test_io_utils.py

```diff
@@ -0,0 +1,57 @@
+import json
+import os
+
+import fsspec
+import pandas as pd
+
+import ecoscope
+
+
+def test_download_file_github_csv():
+    ECOSCOPE_RAW = "https://raw.githubusercontent.com/wildlife-dynamics/ecoscope/master"
+    output_dir = "tests/test_output"
+    ecoscope.io.download_file(
+        f"{ECOSCOPE_RAW}/tests/sample_data/vector/movbank_data.csv",
+        os.path.join(output_dir, "download_data.csv"),
+        overwrite_existing=True,
+    )
+
+    data = pd.read_csv(os.path.join(output_dir, "download_data.csv"))
+    assert len(data) > 0
+
+
+def test_download_file_gdrive():
+    output_dir = "tests/test_output"
+    ecoscope.io.download_file(
+        "https://drive.google.com/uc?export=download&id=1-AQ9_oacUCcAaiZ6SWU77hZWp1oArQw6",
```

Comment:
This url string differs from what a user gets when they click on a GDrive file and select 'Copy Link'. Can we transform from that into what's needed here for downloading? E.g., https://docs.google.com/document/d/1EeBSLv7LOr6maTTVY8T08oD0zbsMb9iUyrAjetIWYUo/edit?usp=sharing

Reply:
done